GVFS/GVFS.Notifications/VFSForGit.Mac/StatusMenuItem/IPC/GVFSTask.m

```diff
@@ -0,0 +1,26 @@
+#import "GVFSTask.h"
+
+@interface GVFSTask()
+
+@property (strong) NSConditionLock *TaskLock;
+@property (strong) NSTask *Task;
+
+@end
+
+// Placeholder class to get VFSForGit info that
+// gets displayed in the Status Item.
+@implementation GVFSTask
+
+- (NSString *) VFSForGitVersion
+{
+    // TODO : Get actual VFSForGit version
```

Comment:
I recommend either addressing these TODOs in this PR or returning an empty string.  

I don't think we want to ship an application that will report its version incorrectly (even if we don't load it by default)

Reply:
Agree. Updated to return empty strings for version.